fendermint/vm/interpreter/src/chain.rs

```diff
@@ -246,7 +246,7 @@ where
         Message = VerifiableMessage,
         DeliverOutput = SignedMessageApplyRes,
         State = FvmExecState<DB>,
-        EndOutput = (PowerUpdates, BlockGasLimit),
+        EndOutput = (PowerUpdates, BlockGasLimit, CurrentValidators),
```

Comment:
If the caller maintains power updates, then there is no need to return `CurrentValidators`? It can be derived?

Reply:
But how would that work during the restarts? Since the cache is not stored?